web/src/pages/Service/Create.tsx

```diff
@@ -71,8 +71,8 @@ const Page: React.FC = (props) => {
     };
 
     const upstreamFormData = upstreamRef.current?.getData();
-    if (upstreamFormData.upstream_id === '') {
-      data.upstream = omit(upstreamFormData, ['upstream_id']);
+    if (!upstreamFormData.upstream_id) {
```

Comment:
Is it possible to have something like:

```js
{
  "upstream_id": "",
  "xxx": ""
}
``` 

in `upstreamFormData`?

Reply:
No, you can only choose an existing upstream or configure your own upstream without upstream_id.
![image](https://user-images.githubusercontent.com/72343596/105438759-950a4b00-5c9e-11eb-8b43-10118c35efcd.png)